index.js

```diff
@@ -41,6 +41,12 @@ var QUESTIONS = [
     }
 ];
 
+var state={
```

Comment:
Using `score` is valid, but it removes the ability to know WHICH question the user answered right or wrong. While not a requirement for this quiz, it's worth thinking about how you might store the HISTORY of user answers and then *compute* the score from that...

i.e.
```
var state = {
  current: 0,
  correctlyAnswered: []
};
```
where `correctlyAnswered` is an array of question indexes that were answered correctly.  Then the length of that array is the "score", plus you could keep track of which were correct, even assign diffrent point values etc.